data/category-payment-ir

```diff
@@ -5,10 +5,12 @@ asanpardakht.net
 behpardakht.com
 iranicard.ir
 mydigipay.com
+pay.ir
 payping.ir
 pec.ir
 pep.co.ir
 sadadpsp.ir
 sep.ir
 shaparak.ir
-zarinpal.ir
+weswap.digital
+zarinpal.com
```

Comment:
please note there should be an empty line at the end of the file.
@alirezax5 @rootmelo92118